docs/website-docs/device-set-up.md

```diff
@@ -0,0 +1,91 @@
+To record a trace, first let’s get you set up.
+
+These preparatory steps only need to be once.
+
+Android
+---------
+
+### 1. Install [PS Companion app](https://play.google.com/store/apps/details?id=com.ps.companion&hl=en_US&gl=US&pli=1) 
+
+You will use PS Companion app later to upload traces and screen recordings to PS Tool.
+
+### 2. Log in with your PS Tool credentials
+
+To log in to PS Companion app, you can use the same credentials you created while registering at productscience.app or use Google authentication.
+
+[Click here](https://productscience.app/recovery-token) if you forgot your credentials.
+
+[Click here](https://productscience.app/sign-up-trial) if you don't have an account yet.
+
+### 3. Make sure tracing is enabled
+Individual phones may have different ways to do this, but these are general guidelines. **Please review these settings even if you are familiar with trace recording on Android, there are some specific settings**
+
+For complete documentation, see the [Official Android Documentation](https://developer.android.com/studio/debug/dev-options)
+
+#### Enable developer options and USB debugging
+
+- Settings &gt; About Phone &gt; Software information &gt; Build Number
+- Tap the Build Number option __7 times__
+
+![enable-developer-settings](../images/dev-options.png)
+
+#### Enable tracing and adjust corresponding settings
+
+- Settings &gt; System &gt; Developer Options &gt; Debugging section &gt; System Tracing
+- Make sure to turn off all options in __Categories__ except 'view: View System' and 'sched: CPU Scheduling'.
+
+![enable-tracing](../images/enable-tracing.png)
+
+- Set __Per-CPU buffer size__ to maximum
+- Enable __Long traces__
+- Enable __Show Quick Settings tile__. This will add Record Trace tile to the Quick Settings panel
```

Comment:
Consider inserting a screenshot of the quikc settings tile with the trace recording icon here.